cmd/kube-apiserver/apiserver.go

```diff
@@ -56,6 +56,9 @@ var (
 		"and it is assumed that port 443 at this address will be proxied/redirected "+
 		"to '-address':'-port'. If blank, the address in the first listed interface "+
 		"will be used.")
+	internalServiceAddress = flag.String("internal_service_address", "10.0.0.20", ""+
```

Comment:
I was going to mention the same thing. GKE, for instance, does not use 10.0.0.0/16 as the block for service IPs.